docs/getting-started/quickstart.md

```diff
@@ -25,47 +25,13 @@ helm repo update
 helm install infra infrahq/infra
 ```
 
-Next, log into your instance of Infra to setup your admin account:
+Next, find the Infra sign-up endpoint:
 
 ```
-infra login localhost
-```
-
-{% callout type="info" %}
-You may be prompted to verify the fingerprint of the server's TLS certificate. The fingerprint can be found in the server logs:
```

Comment:
I removed this long certificates explanation because it didn't seem to fit in a "quickstart" to me, rather it should have its own page. I may have overstepped here though, I can add it back in that case.